### PR TITLE
fixup babel changes

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: .babelrc.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ function preset(_, options = {}) {
         r("babel-preset-env"),
         {
           loose: true,
-          debug: !!options.debug,
+          debug: options.debug,
           modules: "commonjs",
+          useBuiltIns: true,
           targets: {
             node: PRODUCTION ? 4.0 : "current",
-            uglify: PRODUCTION ? true : false,
```
Comment (jquense) [inline .babelrc.js]: since this isn't running the browser code this shouldn't be needed right? it effectively negates everything otherwise

Comment (KyleAMathews) [inline .babelrc.js]: Oh browser code has its own babelrc now? For gatsby-link, etc.?

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 8ba3edb150c2f8fc69c28d288870c4ceae18af93

https://app.netlify.com/sites/using-glamor/deploys/598dc56aa700c4513a2decce

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 202acf337438099c11889c1bd17ce1c863be854c

https://app.netlify.com/sites/using-glamor/deploys/598dc5ba6f4c505290101561

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 31864a3701ca7fbc7775765e304381e4bc6b6d8e

https://app.netlify.com/sites/using-glamor/deploys/598dc950424ef265b09cb04d

**Pull request merged**